### PR TITLE
NET-1159: avoid raising flag for operators that have already been flagged

### PR DESCRIPTION
**Pull request by harbu**
## Limitations and future improvements

- Todo add unit tests (crunch)

## Checklist before requesting a review

- [ ] Is this a breaking change? If it is, be clear in summary.
- [ ] Read through code myself one more time.
- [ ] Make sure any and all `TODO` comments left behind are meant to be left in.
- [ ] Has reasonable passing test coverage?
- [ ] Updated changelog if applicable.
- [ ] Updated documentation if applicable.

### CODE REVIEW

File: packages/broker/src/plugins/operator/ContractFacade.ts [hunk 1]
```diff
@@ -187,6 +195,12 @@ export class ContractFacade {
                                 id
                                 operator {
                                     id
+                                    flagsTargeted(where: {
```
Comment (teogeb) [inline packages/broker/src/plugins/operator/ContractFacade.ts]: We could use hasOpenFlag for the selected node (and therefore no changes to this method)

File: packages/broker/src/plugins/operator/ContractFacade.ts [hunk 2]
```diff
@@ -327,6 +343,33 @@ export class ContractFacade {
...
+        }
+        const queryResult = this.theGraphClient.queryEntities<{ id: string }>(createQuery)
+
+        const flags = await collect(queryResult, 1)
+        if (flags.length > 0) {
+            logger.debug('Found open flag', { flag: flags[0] })
+            return true
+        } else {
+            return false
+        }
```
Comment (teogeb) [inline packages/broker/src/plugins/operator/ContractFacade.ts:361]: Always log the flags? the can simplify to this: `return (flags.length > 0)`

File: packages/broker/src/plugins/operator/ContractFacade.ts (hunk 1 above)
Comment (harbu) [reply packages/broker/src/plugins/operator/ContractFacade.ts]: Simplified to this in a4943bdd3f94fba185979350c37057b5070218de

File: packages/broker/src/plugins/operator/ContractFacade.ts (hunk 2 above)
Comment (harbu) [reply packages/broker/src/plugins/operator/ContractFacade.ts:361]: I'll keep it as it is for now

**Pull request merged**